index.html

```diff
@@ -0,0 +1,92 @@
+                            </svg>
+                            <svg width="25" height="12" viewBox="0 0 25 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <rect opacity="0.35" x="0.5" y="0.833252" width="21" height="10.3333" rx="2.16667" stroke="white"/>
+                                <path opacity="0.4" d="M23 4V8C23.8047 7.66122 24.328 6.87313 24.328 6C24.328 5.12687 23.8047 4.33878 23 4Z" fill="white"/>
+                                <rect x="2" y="2.33325" width="18" height="7.33333" rx="1.33333" fill="white"/>
+                            </svg>
+                                
+                            
+                </div>
+                <div class="bottom-icons">
+                    <svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="white"/>
+                        </svg>
+                        
+                </div>
+                
+                    
+            </div>
+        </nav>
+        <section class="main-body">
+            <div class="container">
+                <div class="intro">
+                    <p>Hey There</p>
+                    <p>I'm Oscar</p>
+                </div>
```

Comment:
Please make sure to put your name inside the `h1` tag because it will make google (and other search engines) knows about the main topic of your website, in this case, you 😄